frontend/src/Judge/components/ShowCard.js

```diff
@@ -0,0 +1,56 @@
+import React from 'react'
+import styled from 'styled-components'
+import { Link } from 'react-router-dom'
+import PropTypes from 'prop-types'
+import Moment from 'react-moment'
+import { Button, Row, Col } from 'reactstrap'
+
+const Card = styled.div`
+  background-color: #f8f9fa;
+  border-radius: 5px;
+  margin-top: 15px;
+  padding: 10px;
+  width: 100%;
+`
+const ButtonContainer = styled.div`
```

Comment:
I would make the start button a little bit bigger since there's so much space on the card itself